Podfile.lock

```diff
@@ -56,4 +56,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 64af6d71574c7a92d01a9446aa874e066917ebe5
 
-COCOAPODS: 1.14.3
+COCOAPODS: 1.15.2
```

Comment:
I updated it as a preliminary, cover-all-bases, step before looking into a spec validation issue I incurred.